scripts/spack/configs/docker/ubuntu22/spack.yaml

```diff
@@ -14,38 +14,25 @@ spack:
     build_stage::
     - $spack/../build_stage
 
-  compilers::
-  - compiler:
-      environment: {}
-      extra_rpaths: []
-      # Axom example sidre_shocktube_F_ex doesn't link otherwise
-      flags:
-        cflags: -fPIC -pthread
-        cxxflags: -fPIC -pthread
-      modules: []
-      operating_system: ubuntu22.04
-      paths:
-        cc: /usr/bin/clang
-        cxx: /usr/bin/clang++
-        f77: /usr/bin/gfortran-11
-        fc: /usr/bin/gfortran-11
-      spec: clang@14.0.0
-      target: x86_64
-  - compiler:
-      environment: {}
-      extra_rpaths: []
-      flags:
-        cflags: -pthread
-        cxxflags: -pthread
-      modules: []
-      operating_system: ubuntu22.04
-      paths:
-        cc: /usr/bin/gcc
-        cxx: /usr/bin/g++
-        f77: /usr/bin/gfortran-13
-        fc: /usr/bin/gfortran-13
-      spec: gcc@13.1.0
-      target: x86_64
+  toolchains:
+    clang_14:
+    - spec: '%c=llvm'
+      when: '%c'
+    - spec: '%cxx=llvm'
+      when: '%cxx'
+    - spec: '%fortran=gcc@11.4.0'
+      when: '%fortran'
+    - spec: '%mpich'
+      when: '%mpi'
```

Comment:
Originally, wanted to bump this Docker container to clang@19 & gcc@13 + Ubuntu 24.04 available through [radiuss-docker](https://github.com/llnl/radiuss-docker/pkgs/container/radiuss/528832165?tag=clang-19-ubuntu-24.04), but ran into problems with the provided MPI causing unit test failures with the `IOManager`/hdf5/etc.... will have to revisit in the future.

Reply:
@chapman39 Not sure if you guys working together can solve this or at least be aware there may be an issue.

Reply:
Issue added: #1676 !